README.md

```diff
@@ -49,12 +49,14 @@ to the `accessories` section. As an example:
         "accessory": "PurpleAirSensor",
         "sensor": "12345",
         "name": "PurpleAir Indoor Sensor",
+        "conversoin": "None",
```

Comment:
```suggestion
        "conversion": "None",
```

Reply:
lol, the AQI is giving me a headache and I can't spell today. Thanks.